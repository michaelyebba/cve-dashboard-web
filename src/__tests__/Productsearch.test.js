import React from 'react';
import {HashRouter} from 'react-router-dom'
import renderer from 'react-test-renderer';
import ProductSearch from '../components/datagrid/Productsearch';

it('renders correctly', () => {
  const tree = renderer
    .create(
        <HashRouter>
            <ProductSearch />
        </HashRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});