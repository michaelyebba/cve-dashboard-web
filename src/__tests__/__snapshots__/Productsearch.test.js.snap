// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <nav
    className="navbar navbar-light bg-light"
  >
    <form
      className="form-inline"
    >
      <input
        className="form-control mr-sm-2"
        placeholder="Search by product"
        type="search"
      />
      <input
        className="btn btn-outline-success my-2 my-sm-0"
        type="button"
        value="Search"
      />
    </form>
  </nav>
</div>
`;
